src/Google/autoload.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$file = __DIR__ . '/../../vendor/autoload.php';
```

Comment:
> I am unsure what that would accomplish.

@bshaffer I think it should work even you use custom `vendor` directory. I may be wrong though.